cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/utils/Utils.java

```diff
@@ -48,4 +50,22 @@ public static void setHostname(final String hostname) {
         Utils.hostname = hostname;
     }
 
+    /**
+     * Get system name.
+     *
+     * @return system name
+     */
+    public static String getSystemname() {
```

Comment:
I am not sure I understand, Utils is used throughout code and it sits in commons. Configuration is there to set it in commons one time during load from yaml.